hw14task1.py

```diff
@@ -0,0 +1,50 @@
+"""
+Homework.
+"""
+# Files.
+# Напишите программу, которая создает текстовый
+# файл(если его нету) "students.txt". Запишите в файл
+# список студентов, номер группы, их оценки. Каждый
+# студент на новой строке. Откройте файл и прочитайте всю
+# информацию из него. Напечатайте общее количество студентов,
+# количество студентов для каждой группы и среднюю оценку для
+# каждой группы. Допишите эту информацию в конец файла.
+# Передусмотрите возможные ошибки и обработайте их.
+
+
+try:
+    with open("students.txt", "w") as file:
```

Comment:
file name still hardcoded

Reply:
If I understand correctly, you need to make sure that the file name can be changed, to do this you need to enter a variable with the file name

Reply:
yes